evm/vm/base.py

```diff
@@ -379,21 +477,21 @@ def get_block_by_header(cls, block_header, db):
         return cls.get_block_class().from_header(block_header, db)
 
     @classmethod
-    def get_prev_headers(cls, last_block_hash, db):
-        prev_headers = []
+    def get_prev_hashes(cls, last_block_hash, db):
+        prev_hashes = []
```

Comment:
Can we use the `to_tuple` decorator here to avoid the `prev_hashes.append` calls.